### ISSUE TITLE
Request for icons: medical icons

### ISSUE THREAD

**Issue by tpanhorst** [ionic-team/ionicons#430]
There are a lot of medical-related apps coming out and Ionic provides a great way to build these apps quickly.  However, the icon set is lacking here.  I have recently needed an ambulance icon and did not have access to one.  I imagine others could make good use of a set of medical icons as well.

Here are some examples from fontawesome.
http://fontawesome.io/icon/ambulance/
http://fontawesome.io/icon/user-md/
http://fontawesome.io/icon/wheelchair/

**Comment by nerdfiles** [comment]
Yeah, seeing "beer" without any mention of "wheelchair," (multiple purposes) or generally "assisted" or "accessibility" kinda rubs me the wrong way. Nothing wrong with combining icon libs, though. FontAwesome and Ionic do complement each other, more or less.

**Comment by Langstra** [comment]
@brandyscarney Is there a list of icons that are going to be added or is there a roadmap of some sort?
